spec/routing/routes_spec.rb

```diff
@@ -4,23 +4,35 @@
   routes { BuriAuth::Engine.routes }
 
   describe 'Custom Routes' do
+    context 'Omniauth' do
+      context 'Facebook' do
+        it '[GET] Handles authentication request' do
+          expect(get: '/oauth/facebook/callback').to route_to(controller: 'buri_auth/omniauth_callbacks', action: 'facebook')
+        end
+
+        it '[POST] Handles authentication request' do
+          expect(post: '/oauth/facebook/callback').to route_to(controller: 'buri_auth/omniauth_callbacks', action: 'facebook')
+        end
+      end
+    end
+
     context 'Registrations' do
       it '[GET] Renders the signup page' do
-        expect(get: '/auth/signup').to route_to(controller: 'devise/registrations', action: 'new')
+        expect(get: '/signup').to route_to(controller: 'devise/registrations', action: 'new')
       end
     end
 
     context 'Sessions' do
       it '[GET] Renders the login page' do
-        expect(get: '/auth/signin').to route_to(controller: 'devise/sessions', action: 'new')
+        expect(get: '/signin').to route_to(controller: 'devise/sessions', action: 'new')
       end
 
       it '[POST] Accepts login requests' do
-        expect(post: '/auth/signin').to route_to(controller: 'devise/sessions', action: 'create')
+        expect(post: '/signin').to route_to(controller: 'devise/sessions', action: 'create')
       end
 
       it '[GET] Accepts logout requests' do
-        expect(get: '/auth/signout').to route_to(controller: 'devise/sessions', action: 'destroy')
+        expect(get: '/signout').to route_to(controller: 'devise/sessions', action: 'destroy')
```

Comment:
How come you could remove the path `auth/`?

Reply:
because we are testing inside the engine, we dont need to prefix the mount route